src/test_report.py

```diff
@@ -27,6 +32,41 @@ def __init__(self, configs, args):
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
         self._logger = Logger("config/logger.conf", "test_report")
+        self._email_host = args.email_host
+        self._email_port = args.email_port
+        self._email_send_from = args.email_send_from
+        self._email_subject = args.email_subject
+        self._email_send_to = args.email_send_to
+
+    def create_email(self, email_content, send_to, send_from, email_subject):
+        email_msg = MIMEMultipart()
+        email_text = email.mime.text.MIMEText(email_content, "plain", "utf-8")
+        email_text.replace_header('Content-Transfer-Encoding', 'quopri')
+        email_text.set_payload(email_content, 'utf-8')
+        email_msg.attach(email_text)
+        if isinstance(send_to, list):
+            email_msg['To'] = ','.join(send_to)
+        else:
+            email_msg['To'] = send_to
+        email_msg['From'] = send_from
+        email_msg['Subject'] = email_subject
+        self._logger.log_message(logging.INFO, email_msg)
+        return email_msg
+
+    def send_email(self, email_msg, email_user, email_password):
+        try:
+            email_server = smtplib.SMTP(self._email_host,
+                                        self._email_port)
+            email_server.starttls()
+            email_server.login(email_user, email_password)
+            email_server.send_message(email_msg)
+        except smtplib.SMTPException as e:
+            self._logger.log_message(logging.ERROR,
+                                     "Email not sent, check your credentials.")
```

Comment:
Thanks for changing the log level to `ERROR`, but I think that the message itself maybe sometimes misleading.
`SMTPException` is a base class for quite a few exceptions and not all of them are credentials related. It'd be good to use the `e` variable that you've already created in the `try` statement to get some proper information about the error and put them in the log.

Please, visit https://docs.python.org/3/library/smtplib.html#smtplib.SMTPException to learn about the errors that inherit after `SMTPException`.